### PR TITLE
Some todos and fixes

### PR DESCRIPTION
**Pull request by mbrandonw**
We have quite a few todos we need to move over from pointfreeco, but these are what we've needed most recently, and this contains a CSS bug fix.

### CODE REVIEW

File: Development.xcconfig [hunk 1]
```diff
@@ -1 +1 @@
-OTHER_SWIFT_FLAGS=$(inherited) -Xfrontend -warn-long-expression-type-checking=80
+OTHER_SWIFT_FLAGS=$(inherited) -Xfrontend -warn-long-expression-type-checking=400
```
Comment (mbrandonw) [inline Development.xcconfig]: bumping to what we use in pointfreeco

File: Sources/Css/Color.swift [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import Prelude
 
-public enum Color: Val, Auto, Inherit, None {
+public enum Color: Val, Inherit, Initial {
```
Comment (mbrandonw) [inline Sources/Css/Color.swift]: turns out `auto` and `none` are not valid colors

File: Sources/Css/Display.swift [hunk 3]
```diff
@@ -214,8 +214,11 @@ public func rect(top: Css.Size, right: Css.Size, bottom: Css.Size, left: Css.Siz
       [
         "rect(",
         top.value().unValue,
+        ",",
```
Comment (mbrandonw) [inline Sources/Css/Display.swift]: forgot to intersperse commas between clip values. it's now rendering as `rect(1px1px1px1px)` causing a css error

File: Sources/HttpPipeline/SignedCookies.swift [hunk 4]
```diff
@@ -105,18 +105,14 @@ extension Response.Header {
   }
 }
 
-private func digest(value: String, secret: String) -> String? {
+public func digest(value: String, secret: String) -> String? {
```
Comment (mbrandonw) [inline Sources/HttpPipeline/SignedCookies.swift]: we've needed some of these encryption/digest methods to be public

File: Development.xcconfig (hunk 1 above)
Comment (stephencelis) [reply Development.xcconfig]: I think everything here is 100ms or under. Would love to not let 400ms stuff sneak in if we can avoid it.

Comment (mbrandonw) [reply Development.xcconfig]: I've got about 20 warnings that are over 100ms :/ it's prob all so dependent on the current state of your computer, but right now pretty much everything is definitely less than 400ms

**Pull request merged**